css/style.css

```diff
@@ -2,19 +2,15 @@ body, html {
     margin: 0;
     padding: 0;
 }
-.container{
-    width: 80%;
-    margin-left: 10%;
+.lb-container{
+    padding: 15px;
+    background-color: rgba(255, 255, 255, .2);
 }
-#list li{
-    list-style: none;
-    margin: 0 auto;
+#images img{
+    margin-top: 15px;
 }
-#list img{
-    width: 100px;
-    position: relative;
-    display: inline-block;
-}
-.imgArray{
+.col-xs-6, .col-md-3, .col-lg-1{
+    float: none;
     display: inline-block;
+    vertical-align: middle;
```

Comment:
I fell right into that one. Sticky note, "DO NOT TOUCH BOOTSTRAP CLASSES". so not just create custom class with those styles and add that class to the parent div of the image.